cmd/tether/main_linux.go

```diff
@@ -38,6 +38,10 @@ func main() {
 		halt()
 	}()
 
+	logFile, _ := os.OpenFile("/dev/ttyS1", os.O_WRONLY|os.O_SYNC, 0644)
+	syscall.Dup2(int(logFile.Fd()), 2)
```

Comment:
Forgot to add also please don't hardcode 2, instead use os.Stderr, while at it you could switch to dup3 as some architectures don't have dup2 :)
```
syscall.Dup3(int(logFile.Fd()), int(os.StdErr.Fd()), 0)
```